.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,39 +16,20 @@ Please read them :pray:
 -->
 
 **Describe the bug**
-<!--
-A clear and concise description of what the bug is.
--->
-
-**Is this a UI bug or a server bug?**
-- [ ] UI
-- [ ] Server
 
-**What is the severity of the bug**
 <!--
-Please select a label for this ticket indicating the severity of this bug.
-A maintainer will add the label to the issue, taking your suggestion into account.
+A clear and concise description of what the bug is.
 -->
 
-- [ ] `severity/Critical`: Weave GitOps is crashing, the UI is inaccessible or a key feature is unusable. There is no known workaround.
```

Comment:
Maybe instead of removing these we can put them inside a comment tag??. So we don't forget what are the rules in choosing one of the severity labels.

Reply:
I agree with @josecordaz - also, leaving it inside a comment tag might _at least_ prompt people to add labels.